tensorboard/components/tf_paginated_view/tf-category-paginated-view.html

```diff
@@ -165,9 +178,16 @@
         border-bottom-right-radius: 0 !important;
       }
 
+      [open-button] .expand-arrow {
+        transform: rotateZ(180deg);
+      }
+
       .name {
         display: inline-flex;
-        float: left;
+        padding: 2px 0;
+        white-space: pre;
+        overflow: hidden;
+        text-overflow: ellipsis;
```

Comment:
would you be able to provide screenshot with this change?

Reply:
I've updated the screenshot in the description to have a case with a long, truncated title name.